deploy/examples/operator.yaml

```diff
@@ -314,6 +314,16 @@ data:
   #      limits:
   #        memory: 256Mi
   #        cpu: 100m
+  # (Optional) Prometheus Rule override default alerts,
+  # Put here all the alerts names and the parameters that you want to override
+  # (will be applied only if monitoring enabled in the cluster)
+  # ROOK_CEPH_MONITORING_PROMETHEUS_RULE: |-
```

Comment:
Why is this in the operator.yaml? The monitoring is cluster-specific, so it needs to be in the cluster CRD

Reply:
I removed the changes from operator.yaml, and moved the customized prometheus to be part of the cluster CRD